pom.xml

```diff
@@ -559,6 +559,13 @@
         <type>pom</type>
         <scope>import</scope>
       </dependency>
+      <dependency>
+        <groupId>com.google.cloud</groupId>
+        <artifactId>google-cloud-pubsublite-bom</artifactId>
+        <version>1.4.0</version>
```

Comment:
This is the latest version as of today: https://search.maven.org/artifact/com.google.cloud/google-cloud-pubsublite-bom